Integrations/MicrosoftTeams/MicrosoftTeams.py

```diff
@@ -789,40 +788,38 @@ def close_channel_request(team_aad_id: str, channel_id: str):
 
 def close_channel():
     """
-    Deletes a mirrored Teams channel
+    Deletes a mirrored Microsoft Teams channel
     """
     integration_context: dict = demisto.getIntegrationContext()
     channel_name: str = demisto.args().get('channel', '')
     channel_id: str = str()
     team_aad_id: str
-    mirrored_channels: list = list()
+    mirrored_channels: list
     if not channel_name:
         # Closing channel as part of autoclose in mirroring process
         investigation: dict = demisto.investigation()
         investigation_id: str = investigation.get('id', '')
         teams: list = json.loads(integration_context.get('teams', '[]'))
-        for team_index, team in enumerate(teams):
+        for team in teams:
             team_aad_id = team.get('team_aad_id', '')
             mirrored_channels = team.get('mirrored_channels', [])
             for channel_index, channel in enumerate(mirrored_channels):
                 if channel.get('investigation_id') == investigation_id:
                     channel_id = channel.get('channel_id', '')
                     close_channel_request(team_aad_id, channel_id)
-                    team_to_update: dict = teams.pop(team_index)
+                    team_to_update: dict = team
                     mirrored_channels.pop(channel_index)
                     team_to_update['mirrored_channels'] = mirrored_channels
-                    teams.append(team_to_update)
                     break
         if not channel_id:
-            raise ValueError('Could not find mirrored Teams channel to close.')
-        integration_context['teams'] = json.dumps(teams)
```

Comment:
revert the deletion of:
`integration_context['teams'] = json.dumps(teams)`
since `teams` is generated from a json string, it will not be updated in the `integration_context` without this line.